### DISCUSSION TITLE
Mac OS support

### DISCUSSION THREAD

**Discussion by Badsheep** [mikebrady/shairport-sync#1948] [Ideas]
Hello,

It's really an "idea", but more a feature request ;)

I love this application but libao is buggy and sometimes I loose the sound... I have to manually reset the sample rate of my soundcard and the sound is back. 

Would you think of the idea of a "real" Mac OS support ?
Thanks you very much anyway! 

Badsheep

**Comment by barracuda156** [comment]
@mikebrady I am somewhat surprised, but looks like there is indeed no macOS audio support? It would be certainly nice to have CoreAudio or at least PortAudio working. Jack or Pulse are a nightmare.

**Comment by Badsheep** [comment]
Hello,

I must say that since an update (like one year ago… I don’t Remember) Shairport is quite stable with libao. But I prefer listening my music with MPD or Navidrome+MPV.
